releasenotes/notes/fix-utils-import-c0f574d76de77261.yaml

```
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Fixes import path to not reference ``__init__``. This could otherwise be a problem for mypy.
```

```suggestion
    Fixes import path to not reference ``__init__``. This could otherwise be a problem for ``mypy``.
```

Spell checking doesn't like `mypy`. Instead of adding to wordlist we can just put in a code block. I can add to the wordlist later.